src/mmw/js/src/app.js

```diff
@@ -122,18 +122,55 @@ function RestAPI() {
 }
 
 function initializeShutterbug() {
+    var googleTileLayerSelector = '#map > .leaflet-google-layer > div > div > div:nth-child(1) > div:nth-child(1)';
+
     $(window)
         .on('shutterbug-saycheese', function() {
             // Set fixed width before screenshot to constrain width to viewport
             $('#model-output-wrapper, body > .map-container').css({
                 'width': window.innerWidth
             });
+
+            var activeBaseLayer = App.getMapView().getActiveBaseLayerName(),
+                googleLayerVisible = activeBaseLayer === 'Satellite with Roads' ||
+                                     activeBaseLayer === 'Terrain';
+
+            if (googleLayerVisible) {
+                // Convert Google Maps CSS Transforms to Left / Right
+                var $googleTileLayer = $(googleTileLayerSelector),
+                    transform = $googleTileLayer.css('transform').split(','),
+                    left = parseFloat(transform[4]),
+                    top = parseFloat(transform[5]);
+
+                $googleTileLayer.css({
+                    transform: 'none',
+                    left: left,
+                    top: top,
+                });
+            }
         })
         .on('shutterbug-asyouwere', function() {
             // Reset after screenshot has been taken
             $('#model-output-wrapper, body > .map-container').css({
                 'width': ''
             });
+
+            var activeBaseLayer = App.getMapView().getActiveBaseLayerName(),
+                googleLayerVisible = activeBaseLayer === 'Satellite with Roads' ||
```

Comment:
Was going to write "Consider using the[ `type` on the layer](https://github.com/WikiWatershed/model-my-watershed/blob/f2c0cbe8177280923a7745da6335c3d07a91b2be/src/mmw/mmw/settings/layer_settings.py#L244)", but when I tried to access it in the console, it looked like it was only available on the mapbox and esri layers. 

Another way we could test if a basemap is google though is to do : 
```
var mapView = App.getMapView(),
      activeLayer = mapView.baseLayers[mapView.getActiveBaseLayerName()],
      googleLayerVisible = !!(activeLayer._google)
```